src/main/java/com/thirdcc/webapp/authorization/ManagementTeamSecurityExpression.java

```diff
@@ -79,6 +102,22 @@ public boolean hasRoleAdminOrIsEventHead(Long eventId) {
         }
     }
 
+    /**
+     * Check if User is Administrator role or Event Head
+     * @param eventId
+     */
+    public boolean isEventHeadOrAdministrator(Long eventId) {
+        return isCurrentAdministrator() || isEventHead(eventId);
+    }
+
+    /**
+     * Check if User is Administrator role or Event Crew
+     * @param eventId
+     */
+    public boolean isEventCrewOrAdministrator(Long eventId) {
+        return isCurrentAdministrator() || isEventCrew(eventId);
+    }
+
```

Comment:
suggest to remove, it's the caller responsibility to use the or statement

```
@PreAuthorize("@managementTeamSecurityExpression.isCurrentCCHead() || @managementTeamSecurityExpression.isEventHead()")
```

resources: https://www.baeldung.com/spring-expression-language

Reply:
resolved in 24b680f2ed9587a7b644cb8051a6fe0babb5615e